.eslintrc

```diff
@@ -3,5 +3,8 @@
 		"@springernature/eslint-config",
 		"@springernature/eslint-config/node",
 		"@springernature/eslint-config/jest"
-	]
+	],
+  "parserOptions": {
+    "ecmaVersion": 9
```

Comment:
Is `ecma 9` supported by `node 8`? Doesn't node 8 not support the rest/spread operators without a flag?

Reply:
> Is ecma 9 supported by node 8

not entirely

>  Doesn't node 8 not support the rest/spread operators without a flag?

on [objects it looks ok](https://node.green/#ES2018-features-object-rest-spread-properties)

lack of async generators and for-await-of makes me a bit sad, but it's workaroundable.

Reply:
yeah it's only on objects, so i get a parse failure from eslint in `bin/index.js` without this

Reply:
I note we have an `engines` entry in the `package.json` but no `.nvmrc` I think? Should we add one?

Reply:
yep can add one

Reply:
The issue with using spread isn't with node, it will work in v8, but eslint with throw a parse error, and there is no way of disabling it either as it isn't a issue with a rule